joomla/joomla-cms
# Feature - Add always count option to 404 redirects

### Summary of Changes
Added a config option for always counting 404 hits - default value is off

### Testing Instructions
Test hit count without applying change, you will see it only counts when the redirect/404 is unpublished. 
Test with changes, the above should still be the same
Test saved with it turned off, above should be the same
Test with it turned on, the count of the 'Redirect' will always increase when hit.

This is a handy feature to track how often the 404 is actually hit still.
